test/python/qpy/test_circuit_load_from_qpy.py

```diff
@@ -325,3 +325,15 @@ def __bool__(self):
                 )
             )
             self.assertEqual(header_data.symbolic_encoding, b"e" if use_symengine else b"p")
+
+
+class TestSymbolExpr(QpyCircuitTestCase):
+    """Test QPY with SymbolExpr"""
+
+    def test_back_slash(self):
+        """Test Parameter with back slash"""
+        qc = QuantumCircuit(2)
+        alpha = Parameter(r"\alpha")
+        beta = Parameter(r"\beta")
+        qc.rz(alpha + beta, 0)
+        self.assert_roundtrip_equal(qc)
```

Comment:
Can you add tests to validate the other fixes you made?

Reply:
I added 2 test cases to test 2 integer fraction add/sub optimization